### ISSUE TITLE
Highlight selected tile/row styles in dropdown menu

### ISSUE THREAD

**Issue by agitator** [plone/plone.app.mosaic#338]
Selected styles should be highlighted within the menu

**Comment by agitator** [comment]
@datakurre @vangheem suggestions how to achieve this?

**Comment by vangheem** [comment]
Will need to make the js that renders the menu look at the current selected row/tile and add a class to it to make it be highlighted.

**Comment by iham** [comment]
i am running a plone 5.2 - the feedback for users is still the same: zero

added a richtext tile, updated format to have a padding-bottom; the tile gets the padding, but the format menu doesn't indicate that this tile i have selected already has the format applied.

makes it hard to get an overview which format to turn off or on.

**Comment by mck9** [comment]
It is still open in 2022 :-) If someone can point me to the direction I could take a look at that.

**Comment by iham** [comment]
@mck9 true ... especially with the formats.

I can't give deeper insight as I don't see how to get some interaction between the selected tile/row and the format menu items to add a class to a menu entry if it is selected.

**Comment by agitator** [comment]
@mck9 yep, would still be very useful - wanna join us at the weekly sprint every Wednesday in the classic-ui channel on discord?

**Comment by mck9** [comment]
> wanna join us at the weekly sprint every Wednesday

@agitator would like to. you can send me an invite. thanks! I must take a deeper look at the code first :-)

**Comment by agitator** [comment]
https://discord.gg/zFY3EBbjaj is the link to Plone on Discord - there's the classic-ui channel text & voice
